e2e/pages/citizen/createCase/C100/otherProceedings/supervisionDocumentUpload.ts

```diff
@@ -112,18 +111,18 @@ export class SupervisionDocumentUploadPage {
 
     // # Not sure if you need to press the button
     // await page.click(
-    //   `${Selectors.button}:text-is("${CareOrderDocumentUploadContent.uploadFile}")`,
+    //   `${Selectors.button}:text-is("${SupervisionDocumentUpload.uploadFile}")`,
```

Comment:
This does need to be pressed, can you also check for the remove button being visible